ci-operator/step-registry/azure/provision/vnet/azure-provision-vnet-commands.sh

```diff
@@ -81,6 +81,11 @@ vnet_addressPrefixes=$(cat "${vnet_info_file}" | jq -r ".[].addressSpace.address
 controlPlaneSubnet=$(cat "${vnet_info_file}" | jq -r ".[].subnets[].name" | grep "master-subnet")
 computeSubnet=$(cat "${vnet_info_file}" | jq -r ".[].subnets[].name" | grep "worker-subnet")
 
+# Store vnet_name, controlPlaneSubnet and computeSubnet to a file to use in the NAT step
```

Comment:
Actually these info is already saved in "${SHARED_DIR}/customer_vnet_subnets.yaml" when running azure/provision/vnet/azure-provision-vnet-commands.sh, so this json file is a bit duplicated. I think we can use the same yaml file to read those needed values.

Reply:
I tried using yq commands to do this, but facing issues. I tried versions `4.22.1` and `3.3.0` but facing errors like `/tmp/yq: line 1: Not: command not found` or `unknown command ".platform.azure.virtualNetwork" for "yq"` Could you help me out how to go about this?

Reply:
Seem like you already resolved it